src/main/proto/options.proto

```diff
@@ -41,17 +41,29 @@ extend google.protobuf.ServiceOptions {
 
   // Globally unique service id
   uint32 id = 50103;
+
+  // Code Access Permission (CAP) Level for the uE
+  uint32 permission_level = 50104;
```

Comment:
CAP spec is not published yet
https://github.com/eclipse-uprotocol/uprotocol-spec/blob/6be703dc287d56582a7493bc997d5db34d122285/basics/permissions.adoc#code-based-access-permissions-caps

Reply:
Yes, it will be soon.